src/pages/index.tsx

```diff
@@ -31,6 +36,38 @@ const Home: NextPage = () => {
             </p>
           </div>
         </Link>
+
+        <button
+          id="theme-toggle"
+          type="button"
+          className="rounded-lg p-2.5 text-sm text-gray-500 hover:bg-gray-100 focus:outline-none focus:ring-4 focus:ring-gray-200 dark:text-gray-400 dark:hover:bg-gray-700 dark:focus:ring-gray-700"
+          onClick={() => {setTheme(theme === "dark"? "light" : "dark")}}
+        >
+          <svg
```

Comment:
Where is the svg being used here? 

![image](https://user-images.githubusercontent.com/89210438/211166029-c04bf39d-3cf4-4846-bab5-f8b212d877f1.png)

Reply:
Should be showing a light theme and dark theme icon, but not being able to. Trying to figure it out.

Reply:
@ssarkar551 I think we should remove the SVG entirely, cuz we will be swapping the design with radix UI later

Reply:
Okay